_docs-sources/guides/working-with-code/versioning.md

```diff
@@ -3,43 +3,73 @@
 ## Versioning
 
 All of the code in the Gruntwork Infrastructure as Code Library is _versioned_. Every time we make a change, we put out a new
-versioned release, and announce it in the monthly
+versioned release, and announce it in the periodical
 [Gruntwork Newsletter](https://blog.gruntwork.io/tagged/gruntwork-newsletter).
 
-![An example of all the versioned updates announced in the monthly Gruntwork Newsletter](/img/guides/stay-up-to-date/newsletter.png)
+![An example of all the versioned updates announced in the Gruntwork Newsletter](/img/guides/stay-up-to-date/newsletter.png)
 
 ### Pinned Versions
 
-When you use the code from the Gruntwork Infrastructure as Code Library, you pin
-yourself to a specific version of the code. That way, you are not accidentally affected by any subsequent changes in
-the Gruntwork Infrastructure as Code Library until you explicitly choose to pull those changes in. And when you do want to pull the
-changes in, it’s just a matter of bumping the version number!
+When you use the code from the Gruntwork IaC Library, you pin yourself to a specific version of the code. That way 
+you are not accidentally affected by any subsequent changes in the Gruntwork IaC Library until you explicitly choose to
+pull those changes in. When you do want to pull the changes in, it’s just a matter of bumping the version number!
 
 ### Semantic Versioning
 
 We use version numbers of the form `MAJOR.MINOR.PATCH` (e.g., `1.2.3`), following the principles of
 _[semantic versioning](https://semver.org)_. In traditional semantic versioning, you increment the:
 
-1. MAJOR version when you make incompatible API changes,
-2. MINOR version when you add functionality in a backwards compatible manner, and
-3. PATCH version when you make backwards compatible bug fixes.
+1. MAJOR version when you make backward incompatible API changes,
+2. MINOR version when you add functionality in a backward compatible manner, and
+3. PATCH version when you make backward compatible bug fixes.
 
-However, much of the Gruntwork Infrastructure as Code Library is still not at version `1.0.0`. Most of the Gruntwork Infrastructure as Code Library is using `0.MINOR.PATCH` version numbers. With `0.MINOR.PATCH`, the rules are a bit different, where you increment the:
+However, much of the Gruntwork IaC Library is still pre-`1.0.0`. Most of the Gruntwork IaC Library uses 
+`0.MINOR.PATCH` version numbers. With `0.MINOR.PATCH`, the rules are a bit different, where you increment the:
 
-1. MINOR version when you make incompatible API changes
-2. PATCH version when you add backwards compatible functionality or bug fixes.
+1. MINOR version when you make backward incompatible API changes, and
+2. PATCH version when you add backward compatible functionality or bug fixes.
+
+### What changes are backward compatible?
```

Comment:
NIT: I found this section a little tough to follow, as I found myself inverting what it said, as what I really wanted to know is, Which changes are backward _incompatible_? So I wonder if its better to define that? E.g., It's backward incompatible if a user has to perform any change other than updating the version number? Changes such as having to update the TF code (e.g., renamed input var) or updating state (e.g., because of a refactor).

Reply:
Ah, good point! I also felt that inversion struggle while writing this.

Reply:
Done in e96e75e